webapp/app/[locale]/get-started/_components/section.tsx

```diff
@@ -16,7 +16,7 @@ export const Section = ({ children, step, ...props }: Props) => (
     <Step {...step} />
     {props.card !== false ? (
       <Card>
-        <div className="flex flex-col gap-y-6 p-4 font-medium md:flex-row md:justify-between md:p-6">
+        <div className="flex flex-col gap-y-6 p-4 font-medium xl:flex-row xl:justify-between xl:p-6">
```

Comment:
This is certainly an improvement 👏🏽 . However, I think `xl` is too large. Let's use `lg` instead

```suggestion
        <div className="flex flex-col gap-y-6 p-4 font-medium lg:flex-row lg:justify-between lg:p-6">
```